source/blog/2016-02-24-devconf-2016.html.md

```diff
@@ -0,0 +1,42 @@
+- Attended [in-depth look of virtual machine migration algorithms - Marcelo Tosatti](https://www.youtube.com/watch?v=XkMIMJKJeTY "in-depth look of virtual machine migration algorithms - Marcelo Tosatti"). I hope to see some of it in coming versions of QEMU, while we work on other improvements to this critical feature.
+- [Growing the ARM server ecosystem - Jim Perrin](https://www.youtube.com/watch?v=q4gU87wdRtI "Growing the ARM server ecosystem - Jim Perrin") was interesting. Seems like a great positive effort, but the road is still long for complete support. Would be cool to support ARM based hosts running virtual machines!
+- Attended [Qemu Disk I/O: Which performs better, Native or Threads? - Pradeep K Surisetty](https://www.youtube.com/watch?v=Jx93riUF5_I "Qemu Disk I/O: Which performs better, Native or Threads? - Pradeep K Surisetty"). The results were not conclusive, though some improvements in native brought it close to or better in some cases to threads. Seems that our default of heuristic for threads in file-based storage and native in block based storage is fine for the time being, but we'll watch closely for developments in this area.
+- Attended [Debugging the Virtualization Layer (libvirt and QEMU) in OpenStack - Kashyap Chamarthy](https://www.youtube.com/watch?v=Dd2AGGMWXQM "Debugging the Virtualization Layer (libvirt and QEMU) in OpenStack - Kashyap Chamarthy") - I think (and asked) if libvirt can do a better job - in saving in a cyclic log the QMP commands sent and the response from the guest and dump it in case of a guest crash.
+- Paid a visit to [Cockpit Hackfest - Dominik Perpeet, Marius Vollmer, Peter Volpe, Stef Walter](https://www.youtube.com/watch?v=TNDe90WSZow "Cockpit Hackfest - Dominik Perpeet, Marius Vollmer, Peter Volpe, Stef Walter") - as we intend to use Cockpit as our UI for oVirt Next Generation Node, it was great to present our use case and exchange thoughts, ideas and directions. Few bugs were filed during the hackfest per our comments.
+- Attended [Dockerizing JBoss Products - David Becvarik](https://www.youtube.com/watch?v=NpyEoFlDzOQ "Dockerizing JBoss Products - David Becvarik"). Great work done by the JBoss team to improve and streamline packaging of JBoss into containers.
+- Attended [Upstream First Testing - Tim Flink](https://www.youtube.com/watch?v=15-yXOJuonQ "Upstream First Testing - Tim Flink"). I discussed with the presenter the possibility to run oVirt with [Lago project on GitHub](https://github.com/lago-project/lago "Lago project on GitHub") in Fedora, in a CI way, by monitoring distgit changes for relevant packages (such as lvm2, device-mapper*, libvirt and others.
+- Obviously attended [High performance VMs in OpenStack - Nikola Dipanov](https://www.youtube.com/watch?v=9J_sEdlpIlQ "High performance VMs in OpenStack - Nikola Dipanov") - looks like oVirt is already doing a lot of what OpenStack are working to achieve high-performance from KVM!
```

Comment:
* Please remove "obviously", not needed.
* "what OpenStack are working" > "what OpenStack is doing"